src/jit/nodeinfo.h

```diff
@@ -32,6 +32,7 @@ class TreeNodeInfo
         isTgtPref             = false;
         regOptional           = false;
         definesAnyRegisters   = false;
+        internalRegDifferentFromTarget = false;
```

Comment:
Done.

---
In reply to: [79938339](https://github.com/dotnet/coreclr/pull/7285#discussion_r79938339) [](ancestors = 79938339)